components/UserProduct.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card class="service-container" :href="product.link">
+  <v-card class="service-container" :href="product.link" :target="product.openInNewTab ? '_blank' : ''">
```

Comment:
We don't want this for all of the products?

Reply:
Just external I think? At least that was what was in the ticket.. I think our internal sites it's possible to get back by taking the dropdown -> account info?